src/mito2/src/sst/parquet/reader.rs

```diff
@@ -467,9 +471,9 @@ impl ParquetReaderBuilder {
         if !self.file_handle.meta_ref().inverted_index_available() {
             return false;
         }
-        let file_size_hint = self.file_handle.meta_ref().inverted_index_size();
+        let file_size_hint = self.file_handle.meta_ref().index_file_size();
```

Comment:
Is it safe to use the puffin file size as this hint? Since we may have multiple different blobs.

Reply:
This hint is passed to the puffin file, not to the reader of the blob